Lib/test/test_sax.py

```diff
@@ -1214,6 +1215,56 @@ def test_expat_incremental_reset(self):
 
         self.assertEqual(result.getvalue(), start + b"<doc>text</doc>")
 
+    def test_flush_reparse_deferral_enabled(self):
+        if pyexpat.version_info < (2, 6, 0):
+            return
```

Comment:
Use the `unittest.skipIf()` decorator or `self.skipTest()` or `raise unittest.SkipTest()`.

Reply:
@serhiy-storchaka good idea, 2x done